packages/bingo/src/cli/setup/runModeSetup.ts

```diff
@@ -156,16 +157,20 @@ export async function runModeSetup<OptionsShape extends AnyShape, Refinements>({
 		};
 	}
 
-	const preparationError = await runSpinnerTask(
-		display,
-		"Preparing local repository",
-		"Prepared local repository",
-		async () => {
-			await createTrackingBranches(remote, system.runner);
-			await createInitialCommit(system.runner, { push: !!remote });
-			await clearLocalGitTags(system.runner);
-		},
-	);
+	let preparationError: Error | void = undefined;
+	const isInGitRepo = await isInGitRepository(system.runner);
+	if (!isInGitRepo) {
+		preparationError = await runSpinnerTask(
```

Comment:
[Cleanup] The using `void` in a type is getting reported by the linter as a violation of [`@typescript-eslint/no-invalid-void-type`](https://typescript-eslint.io/rules/no-invalid-void-type/=). Working around that and streamlining a little bit:

```suggestion
	const preparationError = (await isInGitRepository(system.runner))
		? undefined
		: await runSpinnerTask(
```